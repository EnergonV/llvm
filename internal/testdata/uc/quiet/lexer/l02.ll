; ModuleID = 'l02.c'
target datalayout = "e-m:e-i64:64-f80:128-n8:16:32:64-S128"
target triple = "x86_64-unknown-linux-gnu"

@foo = common global i32 0, align 4
@BarBara = common global i32 0, align 4
@bar_bara = common global i32 0, align 4
@bar4711 = common global i32 0, align 4
@b4rb4r4 = common global i32 0, align 4
@abcdefghijklmnopqrstuvwxyz_ABCDEFGHIJKLMNOPQRSTUVWXYZ_0123456789 = common global i32 0, align 4

; Function Attrs: nounwind uwtable
define i32 @main() #0 {
  ret i32 0
}

attributes #0 = { nounwind uwtable "disable-tail-calls"="false" "less-precise-fpmad"="false" "no-frame-pointer-elim"="true" "no-frame-pointer-elim-non-leaf" "no-infs-fp-math"="false" "no-nans-fp-math"="false" "stack-protector-buffer-size"="8" "target-cpu"="x86-64" "target-features"="+sse,+sse2" "unsafe-fp-math"="false" "use-soft-float"="false" }

!llvm.ident = !{!0}

!0 = !{!"clang version 3.7.1 (tags/RELEASE_371/final)"}
