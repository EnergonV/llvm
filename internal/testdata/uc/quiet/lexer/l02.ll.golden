target datalayout = "e-m:e-i64:64-f80:128-n8:16:32:64-S128"
target triple = "x86_64-unknown-linux-gnu"
@foo = global i32 0
@BarBara = global i32 0
@bar_bara = global i32 0
@bar4711 = global i32 0
@b4rb4r4 = global i32 0
@abcdefghijklmnopqrstuvwxyz_ABCDEFGHIJKLMNOPQRSTUVWXYZ_0123456789 = global i32 0
define i32 @main() {
0:
	ret i32 0
}
